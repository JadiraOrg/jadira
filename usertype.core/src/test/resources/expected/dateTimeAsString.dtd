<!ELEMENT dataset (
    DATETIMEASSTRING*)>

<!ELEMENT DATETIMEASSTRING EMPTY>
<!ATTLIST DATETIMEASSTRING
    ID CDATA #REQUIRED
    MY_DATETIME CDATA #IMPLIED
    MY_OFFSET CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

