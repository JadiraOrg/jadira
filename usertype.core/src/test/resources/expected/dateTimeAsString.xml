<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <dateTimeAsString ID="0" MY_DATETIME="2004-02-25T08:11:10.0" MY_OFFSET="UTC" NAME="test_0"/>
  <dateTimeAsString ID="1" MY_DATETIME="1980-03-01T13:12:11.5" MY_OFFSET="UTC" NAME="test_1"/>
  <dateTimeAsString ID="2" NAME="test_2"/>
</dataset>
